website/source/docs/providers/aws/r/ecs_task_definition.html.markdown

```diff
@@ -24,12 +26,34 @@ resource "aws_ecs_task_definition" "jenkins" {
 }
 ```
 
+### task-definitions/jenkins.json
+
+This is a small subset of the available parameters, see the [AWS docs](http://docs.aws.amazon.com/AmazonECS/latest/developerguide/task_definition_parameters.html) for a full list.
+
+```
+[
+  {
+    "name": "jenkins",
+    "image": "jenkins",
+    "cpu": 10,
+    "memory": 500,
+    "essential": true,
+    "portMappings": [
+      {
+        "containerPort": 80,
+        "hostPort": 80
+      }
+    ]
+  }
+]
+```
+
 ## Argument Reference
 
 The following arguments are supported:
 
 * `family` - (Required) The family, unique name for your task definition.
-* `container_definitions` - (Required) A list of container definitions in JSON format. See [AWS docs](http://docs.aws.amazon.com/AmazonECS/latest/developerguide/task_defintions.html) for syntax.
+* `container_definitions` - (Required) A list of container definitions in JSON format. See [AWS docs](http://docs.aws.amazon.com/AmazonECS/latest/developerguide/create-task-definition.html) for syntax. Note, you only need the containerDefinitions array contents, not the parent hash including the family and volumes keys.
```

Comment:
> containerDefinitions array contents, not

Would you mind changing this to

> containerDefinitions array, not

otherwise people may think that those square brackets may not be necessary.